mobile/src/androidTest/java/ch/epfl/sdp/mobile/test/application/SocialFacadeTest.kt

```diff
@@ -99,4 +100,27 @@ class SocialFacadeTest {
     val userFollowing = user.following.first()
     Truth.assertThat(userFollowing).isEmpty()
   }
+
+  @Test
+  fun get_successfully() = runTest {
```

Comment:
Can you add more information in the name, it's difficult to understand what it's the condition lead to a success